### PR TITLE
Adding HTML export to stories

### PR DESCRIPTION
**Pull request by kiddinn**
Adding support for exporting stories as HTML.

Also adding support for Aggregation groups into export interface.

### CODE REVIEW

File: timesketch/lib/stories/html.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+
+class HTMLStoryExporter(interface.StoryExporter):
+    """HTML story exporter."""
+
+    # String representing the output format of the story.
+    EXPORT_FORMAT = 'html'
+
+    # Number of rows of a DataFrame to include at the top of a markdown table.
+    _DATAFRAM_HEADER_ROWS = 30
```
Comment (berggren) [inline timesketch/lib/stories/html.py:33]: DATAFRAME

Comment (berggren) [reply timesketch/lib/stories/html.py:33]: missing an E

File: timesketch/lib/stories/html.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        return (
+            '{0:s}\n<b>...</b><br/>'
+            '<i>Table contains more records than are displayed. '
+            'Only the first {1:d} records out of total {2:d} are '
+            'printed out. View the full view in Timesketch to get all '
+            'the results.</i>').format(
+                html_code, self._DATAFRAM_HEADER_ROWS, nr_rows)
+
+    def export_story(self):
+        """Export the story as a HTML."""
```
Comment (berggren) [inline timesketch/lib/stories/html.py:66]: -a

Comment (kiddinn) [reply timesketch/lib/stories/html.py:66]: done

**Pull request merged**